content-scope-scripts/content-scope-scripts-impl/src/main/java/com/duckduckgo/contentscopescripts/impl/features/messagebridge/store/MessageBridgeEntity.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.contentscopescripts.impl.features.messagebridge.store
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "message_bridge")
+data class MessageBridgeEntity(
+    @PrimaryKey val id: Int = 1,
```

Comment:
discussed this offline, we don’t really seem to need a Room db, but since this pattern is already used in other c-s-s features let’s not change anything now.